IA-related updates to SEO and metadata content (#169)
mongodb/docs-meta: source/style-guide/seo-guidelines.txt

Search Engine Optimization (SEO) considers the actual terms that users 
enter into search engines (keywords) and employs best practices to 

> improve traffic to web pages from search engines.  Close attention to page metadata and keywords in the page content can improve placement on search results pages.  The following 

Reviewer: ```suggestion
improve traffic to web pages from search engines.  Close attention to page metadata and keywords in the page content can improve page placement in search results.  The following 
```
Suggestion to make more clear!

--------

You can add the actual terms that users enter into search engines 

> (keywords) to the body of your page to improve its SEO.  Google Search Console provides details about the keywords our users enter and can be requested from the IA and SEO teams.  

Reviewer 2: Is there a process for IA and SEO info requests? Something we can link to?

```suggestion
(keywords) to the body of your page to improve its SEO.  Google Search Console provides details about the keywords our users enter and you can request this information from the IA and SEO teams.  
```

------------

The description is a snippet that appears under the link in the search 

> results, which is essential for SEO. Write these descriptions according to 

Reviewer: ```suggestion
results and is essential for SEO. Write these descriptions according to 
```
Suggestion
